packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newBackUpTitleMorphFor..st

```diff
@@ -2,10 +2,9 @@ private - ui
 newBackUpTitleMorphFor: aBPExample
 
 	^ StringMorph new
+		name: #title;
```

Comment:
Why set `name: #title` here as well, when it's already set in `newTitleMorphFor:` in `BPSwimlane`?

Reply:
because when the backup title morph exists that means there is no default title morph